portfolio/src/main/webapp/script.js

```diff
@@ -28,15 +38,45 @@ function addRandomGreeting() {
 }
 
 function showPicture() {
-    var image = document.getElementById("portrait");
-    if (image.style.visibility === "visible") {
-        image.style.visibility = "hidden";
-        image.style.height = "20px";
+    const portrait = document.getElementById("portrait");
+    const pictureButton = document.getElementById("picture-button")
+    if (portrait.style.display === "block") {
+        portrait.style.display = "none";
+        pictureButton.innerHTML = "See me!";
 
     } else {
-        image.style.visibility = "visible"
-        image.style.height = "440px";
-        image.style.width = "600px";
+        portrait.style.display = "block";
+        pictureButton.innerHTML = "Unsee me :(" 
+    }
+}
 
+function showProjects() {
+    const projectContainer = document.getElementById("project-list");
+    
+    if (projectCounter < 3) {
+        console.log("hello");
```

Comment:
remove before submit

Reply:
Just the console log right? Or are you including the other lines too?

Reply:
just console log